src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/VariantEval.java

```diff
@@ -755,7 +755,7 @@ else if (ve instanceof MetricsCollection)
                 metricsCollection.setData(compOverlap.concordantRate, indelSummary.n_SNPs, countVariants.nSNPs, indelSummary.n_indels, multiallelicSummary.nIndels, indelSummary.insertion_to_deletion_ratio, countVariants.insertionDeletionRatio, tiTvVariantEvaluator.tiTvRatio);
         }
 
-        try (PrintStream out = IOUtils.makePrintStreamMaybeGzipped(outFile)){
+        try (PrintStream out = IOUtils.makePrintStreamMaybeGzipped(new GATKPath(outFile.getAbsolutePath()))) {
```

Comment:
Why isn't `outFile` itself converted to `GATKPath` in this tool? (it's still a `File`)

Reply:
Only because changing it causes a cascade of issues. There is another tool that subclasses this one (!) and uses `outFile` with GATKReport. So I was leaving that change for a separate PR that fixes GATKReport.